### ISSUE TITLE
Guidance on Action-Only Fine-tuning on LIBERO using train_internvla.py

### ISSUE THREAD

**Issue by daoshi1593** [InternRobotics/InternVLA-M1#28]
Hi authors,

I am very interested in fine-tuning your pre-trained model on the LIBERO benchmark.

I have located the training script at InternVLA-M1/InternVLA-pruning/training/train_internvla.py. I would like to ask a few specific questions regarding the reproduction process:

Script Usage: Can I use train_internvla.py directly for this purpose?

Action-Only Fine-tuning: I do not need co-training (VLM+VLA). My goal is solely to reproduce the action fine-tuning performance on LIBERO. Could you advise on how to properly configure the yaml/arguments to disable co-training and focus only on the action head?

Dataset Configuration: How should I specify the LIBERO dataset for this specific fine-tuning script? Is there a recommended configuration or data processing step to align with the pre-trained model?

Any guidance or a sample config file for LIBERO fine-tuning would be extremely helpful.

Thanks!

**Comment by MichaelYu781** [comment]
Thanks for your interest!

1. LIBERO fine-tuning entry point
The official launch script for LIBERO fine-tuning is
```
scripts/run_scripts/run_libero_train.sh
```

This is the recommended entry point rather than directly invoking train_internvla.py.

2. Clarification on “co-training”
We would like to clarify what you mean by co-training. It could refer to either:

(a) joint training with VLM data + VLA data, or

(b) whether the VLM backbone is frozen or trainable during fine-tuning.

If you are referring to the second case (freezing the VLM), you can control this by adding the corresponding frozen parameters in the training config, for example here:
https://github.com/InternRobotics/InternVLA-M1/blob/InternVLA-M1/InternVLA/config/training/internvla_cotrain_custom.yaml#L101

3. LIBERO dataset configuration
Please refer to the script mentioned in (1) for how the LIBERO dataset is specified and configured.
Regarding data processing: if you are referring to normalization or related preprocessing steps, these are defined in
https://github.com/InternRobotics/InternVLA-M1/blob/InternVLA-M1/InternVLA/dataloader/gr00t_lerobot/data_config.py

Hope this helps, and feel free to reach out if you have further questions.
